schema/type.go

```diff
@@ -18,6 +18,9 @@ type Type struct {
 	Fields    []*TypeField `json:"fields,omitempty"`
 	TypeExtra `json:",omitempty"`
 	Comments  []string `json:"comments,omitempty"`
+
+	Filename string `json:"-" spew:"-"`
+	Line     int    `json:"-" spew:"-"`
```

Comment:
I'm in favor of removing Filename/Line from webrpc `schema.Type`.

Can we only track Filename/Line in the ridl.Parser?